client/src/app/room/[roomId]/page.tsx

```diff
@@ -32,7 +37,7 @@ const Home = () => {
             }
             // 스크린 크기를 고정값으로 받고 있는데, 반응형으로 받을 수 있는 방법 고려
             const stream = await navigator.mediaDevices.getDisplayMedia({
-                video: true,
+                video: { width: 640, height: 360 },
```

Comment:
```suggestion
                video: true,
```
여기는 혜원님이 true로 바꾸신거라 기존 값으로 바꾸면 될 거 같아요